graphql/types/Subscriptions.graphql

```diff
@@ -1,5 +1,29 @@
+""" Count of Subscription by status """
 type SubscriptionStatusCount {
   active: Int!,
   paused: Int!,
   canceled: Int!
+}
+
+""" Orders generated by subscription """
+type SubscriptionOrder {
+  status: String!,
+  """ SubscriptionId """
+  orderGroup: String,
+  """ SubscriptioOrder creation date """
+  date: String!
+}
+
+""" Count of SubscriptioOrders by status """
+type SubscriptionOrdersStatusCount {
+   triggered: Int!,
+   in_process: Int!,
+   failure: Int!,
+   success: Int!,
+   expired: Int!,
+   order_error: Int!,
```

Comment:
except by this type, all the others graphql types in this repository follow the JS var name conventions (https://www.w3schools.com/js/js_conventions.asp) [camelCase].

Could you create a fieldResolver for rename this field ?

Reply:
👍